src/map/packets/quest_mission_log.cpp

```diff
@@ -85,7 +85,7 @@ CQuestMissionLogPacket::CQuestMissionLogPacket(CCharEntity* PChar, uint8 logID,
                     break;
                 case LOG_CAMPAIGN_TWO:
                     // Second Campaign packet, summoned through logType
-                    generateCampaignMissionPacket(PChar, (uint8)256);
+                    generateCampaignMissionPacket(PChar, 1);
```

Comment:
Assuming we're just enumerating the two packets now? (0,1)

Reply:
That was my thought too